### ISSUE TITLE
on Pycalc-patch-1

### ISSUE THREAD

**Issue by elara04** [DefGamerGreg/Multi-Calculator#9]
i tried to run the code on this branch and the code would not run i found some syntax errors with the code .  i will be going ahead and fixing them and uploading some now operation functions to you list to contribute to this repositopry

**Comment by LoopSoupSnooze** [comment]
Yes, there are some errors, I never had a time to fix the code. Thank you for helping!
